pkg/resources/row_access_policy_acceptance_test.go

```diff
@@ -32,54 +42,393 @@ func TestAcc_RowAccessPolicy(t *testing.T) {
 			{
 				ConfigDirectory: config.TestStepDirectory(),
 				ConfigVariables: m(),
```

Comment:
You can use `ConfigVariablesFromModel` function.

Reply:
+1

Reply:
Done.